### PR TITLE
allow latest simplecov dependency

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by YANOUSHek** [comment]
Can we get this merged? I don't like the fact I have to choose either an older coveralls gem or older simplecov.

**Comment by bastelfreak** [comment]
instead of closing, could we get this merged?...

**Comment by afinetooth** [comment]
Hi @bastelfreak. (Cc: @mrexox)

Hoping this will resolve any questions of support you have for a Ruby-based Coveralls Integration:

**tl;dr**:
Please try [coveralls-ruby-reborn](https://github.com/tagliala/coveralls-ruby-reborn), which is a more up-to-date fork of [coveralls-ruby](https://github.com/lemurheavy/coveralls-ruby) that has been far better maintained and supported over the last few years by [@tagliala](https://github.com/tagliala) (🙏 ). You should be able to just replace the gem reference in your Gemfile.

**Details**:
We (Coveralls team) have been putting our efforts toward the [Coveralls Universal Coverage Reporter](https://github.com/coverallsapp/coverage-reporter), having recently added support for many more [coverage report formats](https://github.com/coverallsapp/coverage-reporter#built-in-support) (which includes Simplecov) and [CI services](https://github.com/coverallsapp/coverage-reporter#supported-ci-services). 

Eventually, the [Reporter](https://github.com/coverallsapp/coverage-reporter) will replace [node-coveralls](https://github.com/nickmerwin/node-coveralls) as the underlying integration behind our two official CI service extensions, the [Coveralls Github Action](https://github.com/marketplace/actions/coveralls-github-action) and [Coveralls Orb for CircleCI](https://circleci.com/developer/orbs/orb/coveralls/coveralls). In fact, [v2 of the Coveralls Github Action](https://github.com/coverallsapp/github-action/releases/tag/v2) with the [Reporter](https://github.com/coverallsapp/coverage-reporter) is now available; as is [v2.1.0 of the Coveralls Orb](https://github.com/coverallsapp/orb/tree/v2.1.0) with [Reporter](https://github.com/coverallsapp/coverage-reporter) as base.

We will continue supporting users of all [Coveralls Language Integrations](https://docs.coveralls.io/) (see left nav of docs), but our long-term goal is to make using Coveralls an easier and more standardized experience, and to make it easier for us to support users.

While we continue providing support for all [Language Integrations](https://docs.coveralls.io/), we will also do our best to update the community on the most up-to-date and well-maintained versions of each (Ex. [Ruby integrations](https://docs.coveralls.io/ruby-and-rails)).
